src/api.ts

```diff
@@ -0,0 +1,14 @@
+const todosUrl = 'https://mate.academy/students-api/todos';
+const usersUrl = 'https://mate.academy/students-api/users/';
```

Comment:
better to use one main url, add all request options when need it

Reply:
agree, do it